### PR TITLE
Refactored landing Page

### PR DESCRIPTION
**Pull request by raghavdhingra**
Screenshot 2021-07-04 at 2 35 36 PM
Screenshot 2021-07-04 at 2 35 47 PM
Screenshot 2021-07-04 at 2 36 52 PM

### CODE REVIEW

**Comment by raghavdhingra** [comment]
Refactored for the responsiveness of the landing page

**Comment by raghavdhingra** [comment]
Will be integrating the responsive Navbar in the next PR

**Comment by sansyrox** [comment]
Amazing work! 🚀 🚀

File: src/components/Home/Home.scss [hunk 1]
```diff
@@ -2,12 +2,9 @@
 
 .home_page {
 	&__container {
-		position: fixed;
 		overflow: auto;
-		height: 100%;
-		width: 100%;
-		margin: auto;
-		text-align: center;
+		height: 100vh;
```
Comment (sansyrox) [inline src/components/Home/Home.scss:6]: This will create an issue in mobile browsers. I will tell you the fix in today's meeting.

**Review by sansyrox — APPROVED** [review]
Great work tho! 🔥

**Pull request merged**